### DISCUSSION TITLE
Discord link on "Troubleshooting" Page is not working

### DISCUSSION THREAD

**Discussion by TausendMedien** [mozilla/hubs#2717] [General Questions & Discussion] [answered]
The link to this discord in the troubleshooting section 
https://hubs.mozilla.com/docs/hubs-troubleshooting.html

is not working: 
discord.gg/wHmY4nd

**Comment by emclaren** [comment, marked as answer]
Thanks for reporting this! I'm doing some work on the docs today - will fix!

**Reply by emclaren** [reply]
Fixed with https://github.com/MozillaReality/hubs-docs/pull/61
